carma_wm/src/WMListenerWorker.cpp

```diff
@@ -198,7 +200,31 @@ void WMListenerWorker::mapUpdateCallback(const autoware_lanelet2_msgs::MapBinPtr
 
   ROS_DEBUG_STREAM("Geofence id" << gf_ptr->id_ << " sends record of intersections size: " << gf_ptr->sim_.intersection_id_to_regem_id_.size());
   if (gf_ptr->sim_.intersection_id_to_regem_id_.size() > 0)
+  {
     world_model_->sim_ = gf_ptr->sim_;
+    for (auto pair : world_model_->sim_.intersection_id_to_regem_id_)
```

Comment:
Seems like these loops are entirely for printing, can they be moved to a separate debug print function to isolate the printing logic from the business logic in the method?

Reply:
factored it out to a different helper function